wrangler-service/src/main/java/co/cask/wrangler/service/directive/DirectivesService.java

```diff
@@ -914,48 +872,31 @@ public void usage(HttpServiceRequest request, HttpServiceResponder responder) {
     try {
       DirectiveConfig config = table.getConfig();
       Map<String, List<String>> aliases = config.getReverseAlias();
-      JsonObject response = new JsonObject();
-      JsonArray values = new JsonArray();
+      List<DirectiveUsage> values = new ArrayList<>();
 
-      int count = 0;
       for (DirectiveInfo directive : composite) {
-        JsonObject usage = new JsonObject();
-        usage.addProperty("directive", directive.name());
-        usage.addProperty("usage", directive.usage());
-        usage.addProperty("description", directive.description());
-        usage.addProperty("excluded", config.isExcluded(directive.name()));
-        usage.addProperty("alias", false);
-        usage.addProperty("scope", directive.scope().name());
-        usage.add("arguments", directive.definition().toJson());
-        usage.add("categories", GSON.toJsonTree(directive.categories()));
-        values.add(usage);
+        DirectiveUsage directiveUsage = new DirectiveUsage(directive.name(), directive.usage(), directive.description(),
+                                                           config.isExcluded(directive.name()), false,
+                                                           directive.scope().name(), directive.definition(),
+                                                           directive.categories());
+        values.add(directiveUsage);
 
         // For this directive we find all aliases and add them to the
         // description.
         if (aliases.containsKey(directive)) {
```

Comment:
Is this comparison correct? `aliases` is a map of `String, List<String>`

Reply:
Good catch, this will always return false. This is an existing bug but I can fix it in this PR